zerver/webhooks/beeminder/doc.md

```diff
@@ -0,0 +1,29 @@
+Get Beeminder notifications in zulip whenever you're going to derail from your goal!
+
+{!create-stream.md!}
+
+{!create-bot-construct-url.md!}
+
+url structure should be like this:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+Modify the parameters of the url above where `api_key` is api key of \
+your Zulip bot and `email` is your zulip email.
+
+* When creating a new goal in beeminder you can check `Make private` option to let \
+bot send you PM otherwise bot will publish in stream.
+
```

Comment:
* Capitalize `beeminder`.
* Change it to `you can check the `Make private` option`.
* Do not use `\` for breaking a line.
* Change it to `to let the bot send you a PM, otherwise, the bot will send a message to a public stream.`